examples/ota-requestor-app/esp32/main/CMakeLists.txt

```diff
@@ -38,7 +38,10 @@ set(SRC_DIRS_LIST
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/descriptor"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/diagnostic-logs-server"
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/ethernet-network-diagnostics-server"
+                      "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src/app/clusters/groups-server"
```

Comment:
Just checking- are there any similar build-related changes that might need to be made for other platforms?

Reply:
CI caught this one. I think it would have failed for other platforms if it was needed.